### ISSUE TITLE
[Fallout 4] Crashes when trying to launch the game via F4SE

### ISSUE THREAD

**Issue by Manic020** [Kaldaien/SpecialK#38]
Even if I have no other mods installed, running f4se_launcher just flashes a black screen real quick before crashing.

**Comment by jarrard** [comment]
Yeah I get this also, and v0.10.1 fails to load f4se fully, it can't see f4se save files for example. Very odd behavior.
